### ISSUE TITLE
v1.6.0 fails to add directory to archive

### ISSUE THREAD

**Issue by CorvusPrudens** [cargo-lambda/cargo-lambda#755]
After the v1.6.0 update, I have been unable to add a directory to the archive with`--include`. This command:

```
cargo lambda deploy --include target/site
```

produces the following erorr:

```
Error:   × failed to add directory `target/site` to zip file
  ╰─▶ invalid Zip archive: Duplicate filename
```

The directory wasn't previously added since the archive is created with this command. This has also worked for previous versions, so it seems to be a regression in v1.6.0.

If more information is required, I can put together a minimal reproduction.

**Comment by calavera** [comment]
Ugh, thanks for opening this issue. I think it should be easy to reproduce with this information, but I’ll let you know what I find.

**Comment by calavera** [comment]
Sorry about the trouble. I've identified the problem it should be solved in #756. I'll release a new version tomorrow morning PT with that patch.

**Comment by CorvusPrudens** [comment]
Oh awesome, thanks!

**Comment by calavera** [comment]
I just released 1.6.1 with the fix: https://github.com/cargo-lambda/cargo-lambda/releases/tag/v1.6.1
